django/contrib/auth/management/commands/changepassword.py

```diff
@@ -63,10 +73,18 @@ def _input_getter_stdin():
         else:
             username = getpass.getuser()
 
+        if not options["interactive"] ^ options["stdin"]:
```

Comment:
I find this clearer, but I know it's subjective
```suggestion
        if (not options["interactive"]) ^ options["stdin"]:
```